kick/http.py

```diff
@@ -236,22 +237,21 @@ async def request(self, route: Route, **kwargs) -> Any:
                 f"Making request to {route.method} {url}. headers: {headers}, params: {kwargs.get('params', None)}, json: {kwargs.get('json', None)}"
             )
             try:
-                res = await self.__session.request(
+                res = await requests_ORI.AsyncSession().request(
```

Comment:
Just like with aiohttp, the session should be reused and stored, not creating a new one